test/extensions/filters/http/custom_response/utility.h

```diff
@@ -123,6 +123,32 @@ constexpr absl::string_view kDefaultConfig = R"EOF(
                   value: "x-bar3"
   )EOF";
 
+constexpr absl::string_view kSinglePredicateConfig = R"EOF(
```

Comment:
nit: seems this is only being used in a single place, so may be better to inline this string in the function